CHANGELOG.md

```diff
@@ -1,7 +1,30 @@
 Magento Functional Testing Framework Changelog
 ================================================
-3.0.0
------
+3.0.0 RC1
+---------
+
+### Enhancements
+* Customizability
```

Comment:
Please also add sub-folders are supported for Test modules.

Reply:
done